### PR TITLE
Add ability to mark/unmark survey as completed

### LINKED ISSUE #108: Allow users to mark if a person has completed a survey.
Currently, the user (surveyor) is not able to mark whether persons under a survey has completed a survey or not. I propose that we add this functionality to support such behaviour.

### PR DESCRIPTION
**Pull request by deepimpactmir**
Resolves #108 

Will add tests in 1.4. There are some changes in the way data is stored in json but I believe it should not affect much.

Please give comments and suggestions!

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Survey.java [hunk 1]
```diff
@@ -50,6 +59,10 @@ public int hashCode() {
      * Format state as text for viewing.
      */
     public String toString() {
-        return '[' + survey + ']';
+        return String.format("[%s, %s]", survey, getStatus());
+    }
+
+    private String getStatus() {
```
Comment (KeithPJX) [inline src/main/java/seedu/address/model/person/Survey.java:65]: Is DONE and NOT_DONE needed? I think the boolean isDone might be enough.

**Review by KeithPJX — APPROVED** [review]
Looks good to me. Thanks for the changes.

File: src/main/java/seedu/address/model/person/Survey.java [hunk 2]
```diff
@@ -38,7 +47,7 @@ public static boolean isValidSurvey(String test) {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Survey // instanceof handles nulls
-                && survey.equals(((Survey) other).survey)); // state check
+                        && survey.equals(((Survey) other).survey)); // state check
```
Comment (KeithPJX) [inline src/main/java/seedu/address/model/person/Survey.java]: I think the equals method should include the check if both surveys are marked as done or not done.

Comment (deepimpactmir) [reply src/main/java/seedu/address/model/person/Survey.java]: Thanks for catching this! Should be fixed now

**Pull request merged**